app/dashboard/admin.py

```diff
@@ -76,13 +76,17 @@ def resend(self, instance):
         return html
 
     def claim(self, instance):
-        if instance.web3_type != 'v2':
+        if instance.web3_type == 'yge':
```

Comment:
Can we consolidate these into one `if` check with `or` since the desired result is the same?

Reply:
i personally think its more readable like this; and the pep standard says that any one line shouldnt be too long